lab.R

```diff
@@ -94,7 +95,7 @@ data("nytcovcounty")
 
 #' 1. *Write a pipe that starts with `nytcovcounty` as input, filters down to California and April 1-August 31, 2020, and assigns the result to a variable `filtered_df`.  Hint: You can compare dates as though they were strings, e.g., `date <= '1980-05-17'` gives you dates on or before May 17, 1980.* 
 filtered_df= nytcovcounty %>% 
-    filter(date < '2020-08-31' & date >'2020-08-01') %>% 
+    filter(date <= '2020-08-31' & date >='2020-08-01') %>% 
```

Comment:
Closer, but this is still just August.  We want April through August.

Reply:
Got it. I misread the question - I thought it asked "1-31 August" for some reason lol